x/datadeal/client/cli/txSellData.go

```diff
@@ -12,9 +12,9 @@ import (
 
 func CmdSellData() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "sell-data [deal_id] [verifiable_cid]",
+		Use:   "sell-data [deal_id] [verifiable_cid] [data_hash]",
```

Comment:
It would be good to inform which encoding scheme to use to be verified by oracles.
Maybe hex encoding?

Reply:
I agree.

Reply:
Thanks for the good comments. Also, I think it would be good to inform which hash function should be used. What hash function should we inform?

Reply:
I think it enough to `SHA256`.
You can refer this.(https://github.com/medibloc/panacea-coracle/blob/main/crypto/hash.go#L8)

Reply:
Agree. `SHA256` + hex encoding is good for here.

Reply:
Thanks, I add it in [5f1b160](https://github.com/medibloc/panacea-core/pull/459/commits/5f1b160d8c5d00ffd2092fc6959e98345f8056ae).